corehq/apps/sms/templates/sms/chat.html

```diff
@@ -25,6 +25,7 @@
         <script src="{% static 'jquery/dist/jquery.min.js' %}"></script>
         <script src="{% static 'hqstyle/js/jquery/jquery-migrate-1.2.1.min.js' %}"></script>
         <script type="text/javascript" src="{% static 'knockout-2.3.0-legacy/knockout.js' %}"></script>
+        <script src="{% static 'style/js/hq.helpers.js' %}"></script>
```

Comment:
You have a point, but I'd rather make the dependencies messier in the short term ( @dimagi/js-team is experimenting with RequireJS to fix our javascript dependency management in general, hopefully making this a moot point soon) if it'll allow koApplyBindings to be used 100% of the time instead of 98% of the time.